_posts/2021-04-19-Time Network Of Letters.md

```diff
@@ -0,0 +1,87 @@
+---
+layout: post
+author: Levi Blinder
+excerpt_separator: <!--more-->
+Title: Time Network Of Letters
+---
+
+[data set](https://docs.google.com/spreadsheets/d/1QALEFpTRY5cKbKZkRepm6U9iEKpJOvoPthxA8qA_uh8/edit?usp=sharing)
```

Comment:
I followed the link and my access was denied! Be sure to update your settings so that readers can access the data. Also, you will probably want to include this link within a sentence or introduction to the effect of, "View the dataset used for this project [here](link for sharing)." Can you also include some information on where you sourced this data? It's important for readers to be able to understand why, how, and where your data was gathered especially.